doc/sphinx/inter.rst

```diff
@@ -449,35 +449,37 @@ implemented currently, which means that the potential is discontinuous
 at :math:`d=d_\mathrm{cut}`. Therefore energy calculations should
 be used with great caution.
 
+.. _Hat interaction:
+
 Hat interaction
 ~~~~~~~~~~~~~~~
 
-.. todo::
-    
-    Not implemented yet.
+.. note::
+    `Feature HAT required.`
+
+The interface for the Lennard-Jones interaction is implemented in 
+:class:`espressomd.interactions.HatInteraction`. The hat parameters
+can be set via::
 
-inter hat
+    system.non_bonded_inter[type1, type2].hat.set_params(**kwargs)
 
-This defines a simple force ramp between particles of the types and .
+This defines a simple force ramp between particle so of two types.
```

Comment:
I don't get what the "[...] particle so of [...]" is supposed to mean.

Reply:
Obviously a typo: change to between particles of two types